tests/src/Mod/Part/App/PartTestHelpers.cpp

```diff
@@ -175,7 +175,7 @@ testing::AssertionResult elementsMatch(const TopoShape& shape,
                              [&, name](const Data::MappedElement& element) {
                                  return matchStringsWithoutClause(element.name.toString(),
                                                                   name,
-                                                                  ";D[a-fA-F0-9]+");
+                                                                  "(;D|;:H|;K)-?[a-fA-F0-9]+");
```

Comment:
Can you amend the comment describing this function to point out what H and K are?

Reply:
Went with inline.